### PR TITLE
LPS-158182 KB Vertical bar should fit whole page

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Built on top of https://github.com/liferay-lima/liferay-portal/pull/3276. Only review last two commits https://github.com/liferay-lima/liferay-portal/pull/3278/commits/f96f4af2e51c53b1a2c29f04fca426d30b0c9121 and https://github.com/liferay-lima/liferay-portal/pull/3278/commits/0c150bf88bbc59292b515972fb3043d72382e256

**Before:** 
Screenshot 2022-07-13 at 17 24 24

**After:** 

Screenshot 2022-07-19 at 12 05 36
Screenshot 2022-07-19 at 12 05 47

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:stop

**Comment by ambrinchaudhary** [comment]
ci:test:sf

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/css/_admin.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import 'atlas-variables';
+
+$controlMenuHeight: 56px;
+
+.knowledge-base-portlet-admin {
+	.kbVerticalBar {
+		> div {
+			height: 100%;
+			min-height: calc(100vh - #{$controlMenuHeight});
```
Comment (boton) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/css/_admin.scss:9]: Why we need `height` and `min-height`?

We always want to fit the whole height.

For me, only height works fine

```scss
height: calc(100vh - #{$controlMenuHeight});
```

But maybe I missing something.

Comment (ambrinchaudhary) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/css/_admin.scss:9]: Sometimes the viewport is shorter that the page content (like in mobile). Similar thing is done with the [info panel](https://github.com/ambrinchaudhary/liferay-portal/blame/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/side_navigation.es.js#L1057-L1057). You can see the difference in the next screenshot (I know the panel overflows the control menu, not sure if it's a component bug, will dig into it)

Only with height: 

height

With min-height: 
min-height

Comment (boton) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/css/_admin.scss:9]: Thanks for the explanation ❤️

**Comment by boton** [comment]
I'm not going to forward it because is on top of https://github.com/liferay-lima/liferay-portal/pull/3276.
But it is fine!

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**